WS2012R2/lisa/xml/STOR_VHDX.xml

```diff
@@ -178,6 +179,22 @@
            <timeout>800</timeout>
        </test>
 
+       <test>
+           <testName>VHDx_DiskHotPlugUnplug_Multi_SCSI</testName>
+           <preTest>
+               <file>setupscripts\AddVhdxHardDisk.ps1</file>
```

Comment:
Hi @Jingli1985 ,
If I run AddVhdxHardDisk.ps1 as preTest script I'm getting these kind of messages: "Remove-VMDvdDrive : The operation cannot be performed while the virtual machine is in its current state." because the VM is running. If we want to keep it like this we need a handle on Get-VMDvdDrive and Remove-VMDvdDrive command.